warehouse/views.py

```diff
@@ -101,6 +101,23 @@ def robotstxt(request):
 
 
 @view_config(
+    route_name="opensearch.xml",
+    renderer="opensearch.xml",
+    decorator=[
+        cache_control(1 * 24 * 60 * 60),         # 1 day
```

Comment:
Instead of calculating these twice, why not have:

```
ONE_DAY = 1 * 24 * 60 * 60
SIX_HOURS = 6 * 60 * 60

@view_config(
    route_name="opensearch.xml",
    renderer="opensearch.xml",
    decorator=[
        cache_control(ONE_DAY),
        origin_cache(ONE_DAY, stale_while_revalidate=SIX_HOURS, stale_if_error=ONE_DAY),
    ]
)
```